src/lib/crypto/kimchi_bindings/stubs/src/pasta_fp_plonk_proof.rs

```diff
@@ -975,8 +969,7 @@ pub fn caml_pasta_fp_plonk_proof_dummy() -> CamlProofWithPublic<CamlGVesta, Caml
     fn comm() -> PolyComm<Vesta> {
         let g = Vesta::prime_subgroup_generator();
         PolyComm {
-            shifted: Some(g),
```

Comment:
I don't know where this is used, but doesn't the behavior change if before it was shift and now isn't?

Reply:
I think it shouldn't -- wherever it's used, removing `shifted` does not seem to break serialization or tests. Will investigate before merging.

Reply:
@fabrizio-m I left a comment in the main PR thread. TLDR I also don't know but it seems to be OK.